frontend/config/env.ts

```diff
@@ -35,7 +35,8 @@ export const GA_URL = process.env.NEXT_PUBLIC_GA_URL;
 
 export const GA_KEY = process.env.NEXT_PUBLIC_GA_KEY;
 
-export const GA_DEBUG = process.env.NEXT_PUBLIC_GA_DEBUG;
+export const GA_DEBUG =
+   process.env.NEXT_PUBLIC_GA_DEBUG?.toLowerCase() === 'true';
```

Comment:
We could add something like this:

```tsx
function getQueryBoolean(
   query: string | string[] | undefined | null,
   options: { considerEmptyAsTrue?: boolean }
): boolean {
   if (query === undefined || query === null) {
      return false;
   }

   if (Array.isArray(query)) {
      return query.some((q) => getQueryBoolean(q, options));
   }

   const boolValue = /true|1/i.test(query);

   const considerEmptyAsTrue = options.considerEmptyAsTrue || true;
   const isEmpty = query === '';

   if (considerEmptyAsTrue && isEmpty) {
       return true;
   } 

   return boolValue;
}
```

Here's some quick tests to show it's behavior:

```

var trueTests = [
'',
'true',
"True",
"TRUE",
"tRuE",
1,
];

console.log(trueTests.map(getQueryBoolean));

var falseTests = [
' ',
'false',
"False",
"FaLSe",
"FALSE",
0,
-2,
'Hello'
];

console.log(falseTests.map(getQueryBoolean));
```

Reply:
I think this would work well for query params but for env vars I think a simple `=== 'true'` check is fine. If we ever change it, I could see switching to 1 vs 0, but the pattern is true/false right now so I'd rather stick to that. Being permissive makes sense in places where we don't control the input, but once you start allowing multiple values (`TRUE`, `true`, `1`) it gets a lot harder to make it strict later.